.github/workflows/ci.yml

```diff
@@ -153,3 +153,13 @@ jobs:
         run: python ./prereqs.py --install
       - name: Run integration tests
         run: python ./build.py --no-check --no-test --wasm --npm --vscode --integration-tests
+
+  runBenchmark:
+    name: run benchmark
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: boa-dev/criterion-compare-action@v3
+        with:
+          branchName: ${{ github.base_ref }}
```

Comment:
Do we want to limit this to run on pull requests only? (Or, what exactly does this do when it runs on a merge group check or manually invoking a workflow?)

Do we want to combine this with the "benches" job?

Reply:
Looking at how the output is summarized and what details are available in the logs, I think this will supersede the existing benches run, so I would be in favor of this PR including a removal of the existing one.